pkg/kopia/command/parse_command_output.go

```diff
@@ -327,6 +331,92 @@ func parseKopiaProgressLine(line string, matchOnlyFinished bool) (stats *Snapsho
 	}
 }
 
+// SnapshotRestoreStats is a container for stats parsed from the output of a `kopia snapshot restore` command.
```

Comment:
```suggestion
// SnapshotRestoreStats is a container for stats parsed from the output of a
// `kopia snapshot restore` command.
```
Wrap text at 80 characters. We have not been consistent but it would be great to do it whenever possible